src/Pages/Staff/StaffList/StaffMember.jsx

```diff
@@ -10,10 +10,7 @@ const StaffMember = ({ member }) => {
 
     return (
         <StaffMemberStyled>
-            <img src={image} />
-            <largeimg src={image} />
-
-            <ModalImage small={img} large={largeimg} alt={name} />
+            <ModalImage small={image} large={image} alt={name} />
```

Comment:
you were just using incorrect variables.

`small={image} large={image}` should reference the `image` prop off of the `member` object. not img, or largeimg.

```js
const { name, image, yearStarted, accolades } = member;
```